services/supervisor/src/mode.rs

```diff
@@ -29,9 +29,11 @@ use exonum_merkledb::access::Access;
 use exonum_proto::ProtobufConvert;
 
 use super::{multisig::MultisigIndex, proto, DeployRequest};
+use failure::{self, format_err};
```

Comment:
We usually (try to) use the "external crates" -> "std" -> "local crate" imports ordering.